orthanc/orthanc-anon/plugin/pixl.py

```diff
@@ -219,18 +181,13 @@ def should_auto_route() -> bool:
 
 def _azure_available() -> bool:
     # Check if AZ_DICOM_ENDPOINT_CLIENT_ID is set
-    return config("AZ_DICOM_ENDPOINT_CLIENT_ID", default="") != ""
+    return bool(config("AZ_DICOM_ENDPOINT_CLIENT_ID", default="") != "")
```

Comment:
At one point it gave me the following mypy error:
`error: Returning Any from function declared to return "bool"  [no-any-return]`
The underlying reason is that `__eq__` doesn't have to return a bool in python (in numpy it could be an array of bools for example).
However, I can only reproduce this error when calling `mypy` explicitly on that file; it doesn't break our `pre-commit` checks or show up in my IDE, so I don't know why I ever encountered it. I think our `mypy` config is still a bit weird.
I will remove it because I don't like it and it seems to not break anything any more.